www/js/diary/services.js

```diff
@@ -16,52 +238,52 @@ angular.module('emission.main.diary.services', ['emission.services'])
     };
 
     timeline.updateFromDatabase = function(day, foundFn, notFoundFn) {
-        console.log("About to show 'Reading from cache'");
-        $ionicLoading.show({
-              template: 'Reading from cache...'
-        });
-        window.cordova.plugins.BEMUserCache.getDocument(getKeyForDate(day),
-          function (tripListArray) {
-             if (tripListArray.length > 0) {
-               var tripListStr = tripListArray[0];
-               var tripList = JSON.parse(tripListStr);
-               console.log("About to hide 'Reading from cache'");
-               $ionicLoading.hide();
-               foundFn(day, tripList);
-             } else {
-               console.log("while reading data for "+day+" from database, no records found");
-               console.log("About to hide 'Reading from cache'");
-               $ionicLoading.hide();
-               notFoundFn(day, "no matching record for key "+getKeyForDate(day));
-             }
-          }, function(error) {
-            console.log("About to hide 'Reading from cache'");
-            $ionicLoading.hide();
-            $ionicPopup.alert({template: JSON.stringify(error)})
-                .then(function(res) {console.log("finished showing alert");});
-        });
-    };
-
-    timeline.updateFromServer = function(day, foundFn, notFoundFn) {
-        console.log("About to show 'Reading from server'");
-        $ionicLoading.show({
-              template: 'Reading from server...'
-        });
-        CommHelper.getTimelineForDay(day, function(response) {
-           var tripList = response.timeline;
-           window.Logger.log(window.Logger.LEVEL_DEBUG,
-                "while reading data for "+day+" from server, got nTrips = "+tripList.length);
-           console.log("About to hide 'Reading from server'");
+      console.log("About to show 'Reading from cache'");
```

Comment:
No I didn't change the factory. I'll reindent in the next pull request. I don't think I can make those changes this week. I'll try to do those next week.

Kris

Sent from my iPhone

> On Apr 3, 2016, at 10:17, shankari <notifications@github.com> wrote:
> 
> In www/js/diary/services.js:
> 
> > -            $ionicPopup.alert({template: JSON.stringify(error)})
> > -                .then(function(res) {console.log("finished showing alert");});
> > -        });
> > -    };
> > -
> > -    timeline.updateFromServer = function(day, foundFn, notFoundFn) {
> > -        console.log("About to show 'Reading from server'");
> > -        $ionicLoading.show({
> > -              template: 'Reading from server...'
> > -        });
> > -        CommHelper.getTimelineForDay(day, function(response) {
> > -           var tripList = response.timeline;
> > -           window.Logger.log(window.Logger.LEVEL_DEBUG,
> > -                "while reading data for "+day+" from server, got nTrips = "+tripList.length);
> > -           console.log("About to hide 'Reading from server'");
> > +      console.log("About to show 'Reading from cache'");
> Are these just indentation changes? You didn't actually change anything in the Timeline factory, did you?
> We should probably have a standard for javascript indentation so that we don't run into issues like this that make it harder to review code.
> 
> What does your editor do? I don't mind following that..
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>